lib/crypto/butil.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
https://www.npmjs.com/package/buf

Allows me to do `db.getToken(buf(token))` that works with either a hex string of a buffer, and then I can `reply({ token: hex(token) })` for the reverse: give me a hex string if its a Buffer, but don't error if it's already a string.